client.js

```diff
@@ -72,12 +72,85 @@
 		readyCallbacks = {},
 		communicators = {};
 
+	var _getOption = function (key, defaultValue) {
+		if(!options) {
+			return defaultValue;
+		}
+		return options[key] || defaultValue;
+	};
+
 	self.fileInputElementId = "siofu_input_"+window.siofu_global.instances++;
 	self.resetFileInputs = true;
-	self.useText = false;
-	self.serializedOctets = false;
-	self.useBuffer = true;
-	self.chunkSize = 1024 * 100; // 100kb default chunk size
+	self.useText = _getOption("useText", false);
+	self.serializedOctets = _getOption("serializedOctets", false);
+	self.useBuffer = _getOption("useBuffer", true);
+	self.chunkSize = _getOption("chunkSize", 1024 * 100); // 100kb default chunk size
+	self.topicName = _getOption("topicName", "siofu");
+
+  /**
+  * Used with wrapOptions to use only one topic for all event instead of using separate topic
+  */
+	self.onlyOneTopic = _getOption("onlyOneTopic", false);
```

Comment:
It seems like it would be a bug to specify option `onlyOneTopic` but not `wrapOptions`.  Combine them into a single option?  Meaning, in _getTopicName, check for the presence of `wrapOptions` instead of `onlyOneTopic`?

Reply:
Done